lib/grape/exceptions/base.rb

```diff
@@ -13,6 +13,7 @@ def initialize(status: nil, message: nil, headers: nil, **_options)
         @status  = status
         @message = message
         @headers = headers
+        super(message)
```

Comment:
Does this duplicate storing `message`, i.e. is `super.to_s` or something of the kind provides the property and we can remove `@message` from our implementation?

Reply:
Makes total sense